src/clj/kixi/hecuba/controller/repl.clj

```diff
@@ -12,23 +12,28 @@
        ([~'type] (submit-item (-> kixi/system :pipeline :head)
                     (merge {:type ~'type}
                            options#)))
+       ([~'type ~'operation] (submit-item (-> kixi/system :pipeline :head)
+                                          (merge {:type ~'type :operation ~'operation}
+                                                 options#)))
        ([] (~name nil)))))
 
-(defreplmethods rollups-> :dest :calculated-datasets :type :rollups)
-(defreplmethods difference-series-> :dest :calculated-datasets :type :difference-series)
-(defreplmethods median-calculation-> :dest :data-quality :type :median-calculation)
-(defreplmethods mislabelled-check-> :dest :data-quality :type :mislabelled-sensors)
-(defreplmethods median-spike-check-> :dest :data-quality :type :spike-check)
-(defreplmethods synthetic-readings-> :dest :calculated-datasets :type :synthetic-readings)
-(defreplmethods resolution-> :dest :data-quality :type :resolution)
-(defreplmethods convert-to-co2-> :dest :calculated-datasets :type :convert-to-co2)
-(defreplmethods convert-to-kwh-> :dest :calculated-datasets :type :convert-to-kwh)
-(defreplmethods sensor-status-check-> :dest :data-quality :type :sensor-status)
-(defreplmethods actual-annual-calculation-> :dest :calculated-fields :type :actual-annual)
+(defreplmethods rollups :dest :calculated-datasets :type :rollups)
+(defreplmethods difference-series :dest :calculated-datasets :type :difference-series)
+(defreplmethods median-calculation :dest :data-quality :type :median-calculation)
+(defreplmethods mislabelled-check :dest :data-quality :type :mislabelled-sensors)
+(defreplmethods median-spike-check :dest :data-quality :type :spike-check)
+(defreplmethods synthetic-readings :dest :calculated-datasets :type :synthetic-readings)
+(defreplmethods resolution :dest :data-quality :type :resolution)
+(defreplmethods convert-to-co2 :dest :calculated-datasets :type :convert-to-co2)
+(defreplmethods convert-to-kwh :dest :calculated-datasets :type :convert-to-kwh)
+(defreplmethods sensor-status-check :dest :data-quality :type :sensor-status)
+(defreplmethods actual-annual-calculation :dest :calculated-fields :type :actual-annual)
+(defreplmethods total-usage-weekly :dest :calculated-datasets :type :total-usage :operation :weekly)
+(defreplmethods total-usage-monthly :dest :calculated-datasets :type :total-usage :operation :weekly)
```

Comment:
We discussed it with @sw1nn some time ago and `->` should be used when the job takes something from one place/format to another. I've initially blindly copied defreplmethods from another project and just replaced the names. This is correcting that mistake.